appendices/assignments.md

```diff
@@ -0,0 +1,197 @@
+
+Each project assignment is described in more detail below.
+
+## First project: Create a package on your own
+
+The first assignment is to create a small, simple package. The *learning
+objective* for this project is that you would practice and reinforce the skills 
+and knowledge you've learned during the course and to start building your
+confidence in package development that will help for the later assignments.
+
+<!--
+NOTE: I moved this due date to after chapter 13 rather than chapter 7, since
+we could keep writing the final exercises to keep building up this package.
+Plus, for those learners who will use this on their own, this would be nice to
+keep going with.
+-->
+
+The project assignment is recommended to be **due** immediately after the
+Package Release lesson (Chapter \@ref(package-release)). At the end of each
+chapter there is a final exercise that gives you tasks to do that guide you in
+completing this first project assignment. These final exercises are used as 
+milestones to ensure you are making progress on your assignment.
+
+An *acceptable* package for completing the assignment would have at a *minimum*:
+
+- Two-three functions.
+- (Optional) a dataset with associated roxygen documentation and an R script that
+generated or extracted the dataset.
+- A unit test for each function.
+- Roxygen documentation for each function.
+- A README with enough detail to know what the package aims to do, a simple
+example for how to use it, how to install it, and how to contribute.
+- A CONTRIBUTING guidelines and a Code of Conduct.
+- Has a passing R CMD check on the Continuous Integration service (in this case
+GitHub Actions).
+
+The instructor will decide the exact details of what is required to complete this
+assignment and what topic or problems the package should focus on.
+For those who are doing this course on their own, the assignment is to create a
+package that should creates some custom summaries of the climate, either local,
+regional, national, or global. These summaries could include minimum and maximum
+historical temperatures, how many days of rainy or cloudy days, or rainfall.
+You can decide on which location to choose, but we recommend creating something
+for your own city or region. Climate data is available from at the
+[World Weather Information Service](https://worldweather.wmo.int/en/dataguide.html)
+website.
+
+For an example of what this might look like, see 
+TODO: Add the repo of the example climate package.
+
+## Second project: Each learner contributing to an existing package
+
+Individual-based, small improvements to existing package: Used as second 
+deliverable, due after "Community involvement". Should only build off of
+and improve an existing package, so doesn't need many milestones. Used to
+reinforce importance of having documentation that can involve the community.
+Could be the final exercises of chapters between Build Management and 
+Community Involvement.
```

Comment:
This reads like a note to self, as opposed to the first section which was instructions for a learner. Will this be revised later or should we polish it a bit now? If now, it should be formatted more like the section starting at line 36.